docs-generator/src/main/scala/GenerateDocs.scala

```diff
@@ -0,0 +1,104 @@
+import java.util
+
+import better.files.File
+import com.codacy.plugins.api._
+import com.codacy.plugins.api.results.Pattern.{Category, Description, Specification}
+import com.codacy.plugins.api.results.Result.Level
+import com.codacy.plugins.api.results.{Pattern, Result, Tool}
+import com.vladsch.flexmark.ast.{Link, Node}
+import com.vladsch.flexmark.ext.tables._
+import com.vladsch.flexmark.parser.Parser
+import com.vladsch.flexmark.util.options.MutableDataSet
+import org.jsoup.Jsoup
+import play.api.libs.json.Json
+import cats.implicits._
+import scala.collection.JavaConverters._
+import scala.reflect.ClassTag
+
+object GenerateDocs {
+  def main(args: Array[String]): Unit = {
+    val file = File(args(0)).contentAsString
+    val dir = args(2)
+    val hadolintRules = parseRulesFile(args(1))
+    parseMarkdownTable(file, dir, hadolintRules)
```

Comment:
Ideally we should split the parse of the table from the write of the files.
If not at least improve the method name.